frontend/src/context/ws-client-provider.tsx

```diff
@@ -93,7 +93,9 @@ export function WsClientProvider({
       messageRateHandler.record(new Date().getTime());
     }
     setEvents((prevEvents) => [...prevEvents, event]);
-    lastEventRef.current = event;
+    if (!Number.isNaN(parseInt(event.id as string, 10))) {
```

Comment:
I don't think this will work - I am taking advantage of the fact that the parseInt function calls `toString` on the value passed in if it is not a string - and sometimes the event id is a number. e.g.:
* `{id: 23, timestamp: "2024-12-05T09:44:59.510897", source: "agent", message: "",…}`

So:
`!Number.isNaN(parseInt(23, 10)) == true`
But:
`(typeof 23 === "string" && !Number.isNaN(parseInt(23, 10))) == false`

Reply:
In that case, nevermind!